lib/SIL/SILVerifier.cpp

```diff
@@ -4302,6 +4302,32 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     }
   }
 
+  void verifyLoadInitialization(SILFunction *F) {
+    llvm::SmallSet<SILValue, 4> AllocInstSet;
+    for (auto &BB : *F) {
+      AllocInstSet.clear();
+      for (auto &I : BB) {
+        if (auto *SI = dyn_cast<StoreInst>(&I)) {
```

Comment:
You don't need to handle StoreInst here because it's covered by the default (else) case below.